odoo/tools/mail.py

```diff
@@ -19,10 +18,26 @@
 from lxml.html import clean, defs
 from werkzeug import urls
 
-import odoo
 from odoo.loglevels import ustr
 from odoo.tools import misc
 
+__all__ = [
+    "email_domain_extract",
+    "email_domain_normalize",
+    "email_normalize",
+    "email_normalize_all",
+    "email_split",
+    "encapsulate_email",
+    "formataddr",
+    "html2plaintext",
+    "html_normalize",
+    "html_sanitize",
+    "is_html_empty",
+    "parse_contact_from_email",
+    "plaintext2html",
+    "single_email_re",
+]
```

Comment:
Should we expose some of these functions as is? These seem to me quite specific to be available directly in the `odoo.tools` namespace:
- `formataddr` change to `def email_format(realname, email_address, charset='utf-8') -> str`
- `single_email_re`  cahnge to `is_single_email(mail: str) -> bool`

Reply:
Ok, I will leave myself a note to take a look after this is merged.